test/services/dumpers/agent-nodejs/agent-nodejs.test.ts

```diff
@@ -0,0 +1,478 @@
+import Context from '@forestadmin/context';
+import appRoot from 'app-root-path';
```

Comment:
The `path` attribute has been removed, especially because of this hack to ease test. 

`path` attribute assignment should not be used (it doesn't do anything actually)

This should have been raised by TS, by typing your `configs` variable.

Reply:
Why do we still use it in other tests ?
`test/services/dumpers/forest-express/dumper-mongo.test.js`
`test/services/dumpers/forest-express/dumper-sequelize.test.js`
`test/services/dumpers/forest-express/dumper-sql.test.js`

Reply:
I removed the path from the test configs, but we still need approot to fetch the created files.
Hope this is what you meant :)
👍